packages/terra-button-group/src/ButtonGroupButton.scss

```diff
@@ -1,14 +1,55 @@
 :local {
   .button-group-button {
+    background-color: var(--terra-button-group-background-color, #fff);
```

Comment:
I'm on the fence of using`--terra-button-group` prefix vs `--terra-button-group-button` prefix.

The `--terra-button-group-button` prefix for these button styles is nice as it frees up the `--terra-button-group` prefix if we ever need to add theming styles to the div all the button-groups sit within and its explicit as to what the themable styles will be applied to. On the flip side, `--terra-button-group-button` is a long prefix and I understand why it was shortened. 

Anyone else have thoughts on this one way or the other?

Reply:
My vote is for `--terra-button-group-button`, it's explicit and would allow custom styles on `--terra-button-group` which might need them in the future

Reply:
It makes sense to use `--terra-button-group-button` as the prefix for the Buttons in the group. Updated: https://github.com/cerner/terra-core/pull/1155/commits/c4e0000d6ceaf96b85053f0046154de1799e7072